src/utils/data.ts

```diff
@@ -0,0 +1,184 @@
+import { useBenchRunStore } from '../stores/benchRunStore'
+import { useCacheFileStore } from '../stores/cacheFileStore'
+import { useCommitStore } from '../stores/commitStore'
+import { useDataPointStore } from '../stores/dataPointsStore'
+import { useLabelStore } from '../stores/labelStore'
+
+export interface Commit {
```

Comment:
These `interface`s seem to describe the structure of JSON files that are being loaded in. How are they connected to the schema definitions? Why do we have TypeScript definition and also JSON schema definition?

Having two independent definition is good only for one thing: to make them inconsistent.

Reply:
Let's deal with this schema-related issues once we tackle the new data in https://github.com/enso-org/enso/pull/10224